AssemblerLib/LocalDataInitializer.h

```diff
@@ -240,18 +242,19 @@ class LocalDataInitializer
 
     /// Sets the provided data_ptr to the newly created local assembler data.
     void operator()(
-            const MeshLib::Element& e,
+            std::size_t const mesh_item_id,
```

Comment:
Please add the previous comment regarding this id, which was:
```
/// The positions in the global matrix/vector are taken from
/// the LocalToGlobalIndexMap provided in the constructor at index \c id.
/// \attention The index \c id is not necesserily the mesh item's id.
```

The reason for not being the mesh element's id is more clear when we have two meshes. Then the second's mesh element ids are offset by first mesh size. They correspond to the entries in the dof table. So one could also call it dof_table_entry_id or sth. along this lines.